src/DataSet/ArrayDataSet.php

```diff
@@ -14,11 +16,16 @@ public function __construct(private array $data = [])
 
     public function getAttributeValue(string $attribute): mixed
     {
-        return $this->data[$attribute] ?? null;
+        return $this->hasAttribute($attribute) ? $this->data[$attribute] : null;
```

Comment:
```suggestion
        return $this->data[$attribute] ?? null;
```

Previous code is more elegant.

Reply:
Yes, using `hasAttribute()` outside of this scope is enough. Adjusted in other places as well.